inexor/engine/world.cpp

```diff
@@ -548,6 +548,7 @@ void entselectionbox(const entity &e, vec &eo, vec &es)
 VAR(entselsnap, 0, 0, 1);
 VAR(entmovingshadow, 0, 1, 1);
 
+
```

Comment:
Ahh, my bad. I have already made a pull request, not too sure how I would discard a change inside a file after doing so.